components/qma6100p/idf_component.yml

```
@@ -1,6 +1,6 @@
-version: "1.0.0"
+version: "2.0.0"
 description: I2C driver for QMA6100P accelerometer
 url: https://github.com/espressif/esp-bsp/tree/master/components/qma6100p
 dependencies:
-  idf : ">=4.0"
+  idf : ">=5.4"
```

```suggestion
  idf : ">=5.2"
```

Driver-NG was added in idf v5.2